README.rst

```diff
@@ -814,6 +814,7 @@ automated builds of different OpenWISP modules.
 ^^^^^^^^^^^^^^^^^^^^^^
 
 Shell script to automatically format Python code. It runs ``isort`` and ``black``.
+It also runs ``stylelint`` to format CSS code.
```

Comment:
Shell script to automatically format Python code. It runs ``isort``. ``black`` and  ``stylelint``.

Reply:
I think its better. 
Shell script to automatically format Python and CSS code. It runs isort. black and stylelint.

Reply:
Please mention that the command won't fail if ``stylelint`` is not installed but will skip instead.